source/userguide/entity_recognizer.rst

```diff
@@ -28,6 +28,11 @@ Entities in MindMeld are categorized into two types:
 
 This chapter focuses on training entity recognition models for detecting all the custom entities used by your app.
 
+.. note::
+   - Currently MindMeld uses a fork of Duckling for system entity recognition. However, MindMeld Duckling binaries are only available in certain operating systems.
```

Comment:
Specify which OS are supported

Reply:
Do we have the list somewhere? I can't find it in the documentation.

Reply:
"ubuntu-16.04"
    "ubuntu-18.04"
    "darwin"

Reply:
done